mlflow/store/artifact/artifact_repo.py

```diff
@@ -265,11 +267,13 @@ def _download_file(src_artifact_path, dst_local_dir_path):
                 except Exception as e:
                     path = futures[f]
                     failed_downloads[path] = e
+                    tracebacks[path] = traceback.format_exc()
 
         if failed_downloads:
-            template = "##### File {path} #####\n{error}"
+            template = "##### File {path} #####\n{error}\nTraceback:\n{traceback}\n"
```

Comment:
Nice, we should check with OSS folks, but we may want to include the tracebacks as optional debug-level logging to start

Reply:
Since it's an error message, I check if the logger has `DEBUG` level enabled before including the Traceback or not.